### DISCUSSION TITLE
Omnetpp and IEEE 802.11p

### DISCUSSION THREAD

**Discussion by Aravindhan-Mathivanan** [omnetpp/omnetpp#1211] [Q&A] [answered]
Hi everyone, I'm new to Vehicular communication , for my project i have to implement V2I communication using IEEE 802.11p protocol and couple it with SUMO simulation, i have no prior knowledge with both of these software i don't know where to start from. So I need some guidance from where to start with for learning omnetpp

**Comment by rhornig** [comment, marked as answer]
Veins is your friend. https://veins.car2x.org/
